.github/ISSUE_TEMPLATE/release.md

```diff
@@ -24,18 +23,23 @@ candidate branch.
     runtime logic.
 - [ ] Check for any upstream storage migrations and perform tests with `try-runtime`, if any.
 
-The following checks can be performed after we have forked off to the release branch.
+The following checks can be performed after we have frozen our release candidate:
+
+- [ ] Code freeze should typically happen one week prior to release, to ensure we have enough time for related testing.
```

Comment:
I'd say "Notify everyone, especially people with merge rights to `manta` (stechu, Dengjianping , ghzlatarev) that a release is ongoing and no more merges to `manta` should happen until told otherwise."

More specific than just "code freeze"